### ISSUE TITLE
Source Compiled Caja 1.28 doesn't display buttons or allow right clicks and more. Earlier versions don't have this problem

### ISSUE THREAD

**Issue by LinuxBeaver** [mate-desktop/caja#1768]
I am on Fedora 38 and compiling Caja was a success but a lot is missing. I guess this Caja is not integrated into my system because the Caja from my systems repo works fine. However it does remember my bookmarks.  Alt back and forward do not allow me to go back forward or home. 

![image](https://github.com/mate-desktop/caja/assets/78667207/4e35ffc9-e75c-4b61-9b6d-43ebf3521a08)

**Comment by LinuxBeaver** [comment]
Caja 1.26.1 was  compiled successfully (the same version my system uses) without any problem and I renamed my systems caja to `/usr/bin/caja2` to make sure it wasn't calling that.

btw I am running Caja on the Cinnamon Desktop so that might explain things.
![image](https://github.com/mate-desktop/caja/assets/78667207/4c7665d4-1a94-4507-82c1-6d5ff15f516e)

**Comment by LinuxBeaver** [comment]
![image](https://github.com/mate-desktop/caja/assets/78667207/12e9244e-11f4-4516-9076-b663e67b7cba)

I just compiled  Caja 1.26.3 and it doesn't have the problem 1.28.0 has

**Comment by LinuxBeaver** [comment]
I just recompiled 1.28.0 and now it works 100% TMK. I think it went wrong because of an original compile failure and I didn't clean up the failed build. 

![image](https://github.com/mate-desktop/caja/assets/78667207/06eedacc-55c9-48c6-8d64-30206903c0b6)


These are the commands I use to compile Caja 

`./configure`

`./autogen.sh --prefix=/usr
make`

While building 1.28.0 15-20 minutes ago I made errors and didn't clean. 


This issue can be closed as I solved my own problem.
